spiffworkflow-backend/src/spiffworkflow_backend/config/default.py

```diff
@@ -2,6 +2,8 @@
 import re
 from os import environ
 
+FLASK_SESSION_SECRET_KEY = environ.get("FLASK_SESSION_SECRET_KEY")
```

Comment:
Should we prepend with "SPIFFWORKFLOW_BACKEND_" here? If flask doesn't use it directly then it may be a good idea and looks like there my be other flask env vars we should namespace as well.